usmqe/gluster/gluster.py

```diff
@@ -139,22 +139,14 @@ def find_volume_name(self, name, expected=True):
         volume_name = self.get_volume_names()
         found = False
         # TODO test what vol_name looks like when there are more volumes name
-        if isinstance(volume_name, list):
-            for item in volume_name:
-                # TODO use pytest.check in if?
-                if item == name:
-                    found = True
-        else:
-            if volume_name == name:
-                found = True
         if expected:
             pytest.check(
-                found,
+                name in volume_name if isinstance(volume_name, list) else volume_name == name,
                 "{} should be among volumes from output \
                 of gluster volume info command".format(name))
         else:
             pytest.check(
-                not found,
+                name not in volume_name if isinstance(volume_name, list) else volume_name != name,
                 "{} should not be among volumes from output \
                 of gluster volume info command".format(name))
         return found
```

Comment:
I am really sorry, I have several doubts about this module.
1. GlusterVolume init parameter should not be cluster but volume_name and it will be used only for GlusterVolume not for GlusterCommon. self.cluster is not used anyway.
2. the info method doesn't return dictionary and for sure not output of the 'gluster --format json --cluster CLUSTERNAME status' command
3. info should work with the volume name 'gluster volume info <volume_name>'
4. GlusterVolume could remember not just volume_name, but also it could have other parameters which will be populated after info method call. It could also be one dictionary with parameter names as keys.